### PR TITLE
Update RU translation, part 2

### PR DESCRIPTION
**Pull request by AAS**
Update previous strings based on critique from Iniquitatis
Changed some "е" to "ё" where it is applicable, not all of them found yet.
Resolve some inconsistencies in characters naming.
Add translations to some quest items, monsters, options, etc.

### CODE REVIEW

**Comment by AJenbo** [comment]
Note that you have multiple conflicts, make sure you pull the latest master an resolve possible conflicts.

**Comment by AAS** [comment]
> Note that you have multiple conflicts, make sure you pull the latest master an resolve possible conflicts.

Done. 

Also russian translations is full of purposely made mistakes.

For example commit 2cf6bbc45197d9fdf527b3b8bfd22d1b8fd355d4, where "The boy Wirt has a blackness upon his soul," string translated correctly, but appended with "but he's yummy inside)" russian text.

Someone sane should review this things before merging.

**Comment by AJenbo** [comment]
> Someone sane should review this things before merging.

Hmm we had a suspicion that this might be going on. Thanks for confirming.

File: Translations/ru.po [hunk 1]
```diff
@@ -564,8 +562,8 @@ msgid ""
 "greater challenge. This is recommended for experienced characters only."
 msgstr ""
 "Быстрая скорость\n"
-"Теперь обитатели лабиринта стали быстрее. Эта скорость рекомендуется для "
-"опытных персонажей."
+"Теперь обитатели лабиринта стали быстрее и будут большим вызовом. Эта "
```
Comment (Iniquitatis) [inline Translations/ru.po:565]: Может, "большим испытанием"? "... будут большим вызовом" звучит не совсем естественно.

File: Translations/ru.po [hunk 2]
```diff
@@ -574,17 +572,17 @@ msgid ""
 "Only an experienced champion should try their luck at this speed."
 msgstr ""
 "Очень быстрая скорость\n"
-"Большинство монстров найдут вас быстрее чем вы найдете пульт от телевизора. "
-"Только опытные чемпионы смогут испытать свои навыки в скорости."
+"Большинство монстров найдут вас быстрее чем раньше. Только опытные чемпионы "
```
Comment (Iniquitatis) [inline Translations/ru.po:575]: Сравнительный оборот: "быстрее, чем раньше".

File: Translations/ru.po [hunk 3]
```diff
@@ -1264,281 +1262,281 @@ msgstr ""
...
 
-#: Source/error.cpp:75
+#: Source/error.cpp:76
 msgid "That which can break will."
 msgstr ""
 
 #: Source/gamemenu.cpp:38
 msgid "Save Game"
-msgstr ""
+msgstr "Сохранить Игру"
```
Comment (Iniquitatis) [inline Translations/ru.po:1483]: Кстати, спорный момент. Не уверен, стоит ли использовать Title Case в русской локализации.

File: Translations/ru.po [hunk 4]
```diff
@@ -1264,281 +1262,281 @@ msgstr ""
...
 
 #: Source/gamemenu.cpp:64
 msgid "Previous Menu"
-msgstr ""
+msgstr "Предыдущее Меню"
 
 #: Source/gamemenu.cpp:71
 msgid "Music Disabled"
-msgstr ""
+msgstr "Музыка выключена"
```
Comment (Iniquitatis) [inline Translations/ru.po:1511]: "Музыка выключена", но "Звук Выключен" (заглавная буква). Одно из двух.

File: Translations/ru.po [hunk 5]
```diff
@@ -1264,281 +1262,281 @@ msgstr ""
...
 
 #: Source/gamemenu.cpp:76
 msgid "Sound Disabled"
-msgstr ""
+msgstr "Звук Выключен"
 
 #: Source/gamemenu.cpp:166
 msgid "Speed: Fastest"
-msgstr ""
+msgstr "Скорость: Сверхбыстрая"
```
Comment (Iniquitatis) [inline Translations/ru.po:1523]: Уже ведь меняли "сверхбыстрая" на "самая быстрая".

File: Translations/ru.po [hunk 6]
```diff
@@ -5400,219 +5400,221 @@ msgstr ""
...
 
 #: Source/objects.cpp:255
 msgid "The Tale of the Horadrim"
-msgstr ""
+msgstr "Сказ Хорадримов"
 
 #: Source/objects.cpp:256
 msgid "The Dark Exile"
-msgstr ""
+msgstr "Темный изгнанник"
```
Comment (Iniquitatis) [inline Translations/ru.po:5416]: "Т**ё**мный".

File: Translations/ru.po [hunk 7]
```diff
@@ -5400,219 +5400,221 @@ msgstr ""
...
 
 #: Source/objects.cpp:257
 msgid "The Sin War"
-msgstr ""
+msgstr "Война Грехов"
 
 #: Source/objects.cpp:258
 msgid "The Binding of the Three"
-msgstr ""
+msgstr "Связь Трех"
```
Comment (Iniquitatis) [inline Translations/ru.po:5424]: "Тр**ё**х".

File: Translations/ru.po [hunk 8]
```diff
@@ -5400,219 +5400,221 @@ msgstr ""
...
+#: Source/objects.cpp:5542
 msgid "Cauldron"
-msgstr ""
+msgstr "Котел"
 
-#: Source/objects.cpp:5543
+#: Source/objects.cpp:5545
 msgid "Murky Pool"
-msgstr ""
+msgstr "Темный бассейн"
```
Comment (Iniquitatis) [inline Translations/ru.po:5586]: "Т**ё**мный".

File: Translations/ru.po [hunk 9]
```diff
@@ -5400,219 +5400,221 @@ msgstr ""
...
 
-#: Source/objects.cpp:5546
+#: Source/objects.cpp:5548
 msgid "Fountain of Tears"
-msgstr ""
+msgstr "Фонтан слез"
```
Comment (Iniquitatis) [inline Translations/ru.po:5590]: "Сл**ё**з".

File: Translations/ru.po [hunk 10]
```diff
@@ -5669,106 +5671,108 @@ msgstr ""
...
+#: Source/quests.cpp:55
 msgid "Archbishop Lazarus"
-msgstr ""
+msgstr "Архиепископ Лазарь"
 
-#: Source/quests.cpp:55
+#: Source/quests.cpp:56
 msgid "Grave Matters"
-msgstr ""
+msgstr "Серьезные вопросы"
```
Comment (Iniquitatis) [inline Translations/ru.po:5712]: "Серь**ё**зный".

File: Translations/ru.po [hunk 11]
```diff
@@ -7031,13 +7035,10 @@ msgid ""
 " \n"
 "What? Oh, yes... uh, well, I suppose you could see what someone else knows. |"
 msgstr ""
-"Вы читаете интересный стишок, написанный в стиле, который напоминает мне "
-"другие произведения. Дайте мне подумать - что это было?\n"
+"Я никогда не интересовался поэзией. Иногда, я нанимал менестрелей, когда "
```
Comment (Iniquitatis) [inline Translations/ru.po:7038]: Запятая после "иногда" не нужна.

**Comment by AJenbo** [comment]
Damn I really gotta let tease sit for a bit before merging :(

File: Translations/ru.po [hunk 12]
```diff
@@ -7773,16 +7776,16 @@ msgid ""
 "your eyes open for a black mushroom. It should be fairly large and easy to "
 "identify. If you find it, bring it to me, won't you? |"
 msgstr ""
-"Что же мы имеем? Интересно, это похоже на книгу с реагентами. Ищите за "
-"черным грибом. Он должен быть довольно большим и легко распознаваемым. Если "
-"найдешь его то принесёшь мне, правда? |"
+"Что у нас тут? Интересно, похоже на книгу реагентов. Поищи чёрный гриб. Он "
+"должен быть довольно большим и легко распознаваемым. Если найдёшь его то "
```
Comment (Iniquitatis) [inline Translations/ru.po:7780]: Запятая перед "то".

File: Translations/ru.po [hunk 13]
```diff
@@ -9084,21 +9086,21 @@ msgid ""
 msgstr ""
 "Будьте внимательны и изучайте истины, которые здесь лежат, поскольку они - "
 "последнее из наследия Хорадримов. Почти триста лет назад стало известно, что "
-"Три главных зла Пылающего ада таинственным образом пришли в наш мир. Три "
-"брата десятилетиями опустошали земли востока, в то время как человечество "
-"трепетало перед ними. Наш Орден - Хорадрим - был основан группой скрытных "
-"магов, чтобы раз и навсегда выследить и схватить Три Зла.\n"
+"Три главных зла Пылающих Преисподней таинственным образом пришли в наш мир. "
```
Comment (Iniquitatis) [inline Translations/ru.po:9089]: "Пылающей".

File: Translations/ru.po [hunk 14]
```diff
@@ -9084,21 +9086,21 @@ msgid ""
...
+"Три брата десятилетиями опустошали земли востока, в то время как "
+"человечество трепетало перед ними. Наш Орден - Хорадримом - был основан "
+"группой скрытных магов, чтобы раз и навсегда выследить и схватить Три Зла.\n"
 " \n"
-"Первоначальный Хорадрим захватил два из Трех зол внутри могущественных "
+"Первоначальный Хорадримы поймали два из Трёх зол внутри могущественных "
```
Comment (Iniquitatis) [inline Translations/ru.po:9094]: "Первоначальный Хорадрим" или "первоначальные Хорадримы".

Возможно, "поймали двоих".

File: Translations/ru.po [hunk 15]
```diff
@@ -9784,15 +9785,15 @@ msgid ""
...
+"в этих стенах. Моё время кончилось. Даже сейчас его адские приспешники "
 "царапают хлипкую дверь, за которой я прячусь. \n"
 "\n"
 "Я поймал демона с помощью тайной магии и заключил его в огромные стены, но, "
 "боюсь, этого будет недостаточно.\n"
 "\n"
-"Заклинания из трех моих гримуаров обеспечат вам безопасный вход в его "
-"владения, но только если оно будут накастованы в правильной "
+"Заклинания из трёх моих гримуаров обеспечат вам безопасный вход в его "
+"владения, но только если оно будут произнесены в правильной "
```
Comment (Iniquitatis) [inline Translations/ru.po:9796]: "Они будут произнесены".

**Pull request merged**